src/content/general/organizations/index.md

```diff
@@ -0,0 +1,143 @@
+## Naming conventions {#naming-conventions}
+
+Organization names (technically: organization CR names) must follow these rules:
+
+- Must be unique within the management cluster
+- Must contain at most 59 characters.
+- Has to conform the same convention as Kubernetes namespaces additionally (i. e. the [DNS label names convention](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)), which means:
+    - contain only lowercase alphanumeric characters or '-'
+    - start with an alphanumeric character
+    - end with an alphanumeric character
+
+Since there will be a namespace created for each org, prefixed with `org-`, we recommend against using that same prefix in the organization name, to avoid confusion.
+
+## Organization namespace {#namespace}
+
+For each organization there is a namespace in the management cluster. The namespace is named after the organization CR name, prefixed with `org-`.
+
+For example, for an organization `acme`, there is the defining organization CR named `acme`. In addition, cluster-operator ensures the existence of the namespace `org-acme` in the management cluster.
+
+We recommend to place all resources to be owned by an organization into the organization's namespace. Our user interfaces will work towards supporting this as a default.
+
+### Namespace use in different providers {#namespace-use}
+
+Giant Swarm is currently working towards making the organization's namespace the default namespace for all resources owned by the organization: clusters, node pools, apps, and more. We have (as of April 2021) reached different stages on different providers.
+
+#### Azure {#namespace-use-azure}
+
+With the latest workload cluster releases for Azure (as of April 2021 that is v14.1.x), all cluster and node pool resources are placed in the organization namespace by default.
+
+Resources belonging to apps deployed to workload clusters are not yet placed in the organization namespace.
+
+#### AWS {#namespace-use-aws}
+
+With the latest AWS releases (as of April 2021 that's v14.1.x), the organization namespace is not yet used by default.
+
+#### On-premises (KVM) {#namespace-use-onprem}
+
+With the latest KVM releases (as of April 2021 that's v13.1.x), the organization namespace is not yet used by default.
+
+## Access control
+
+The management API relies on single sign-on using each customer's own identity provider for authentication.
+
+As the customer's admin for an installation, you decide which users should get access to an organization's resources. You do so associating the user or group identifiers from your own identity provider with permissions for resources in the organization's namespace. All of this is done using standard Kubernetes RBAC elements:
+
+- `Role` and `ClusterRole` resources define the permissions to be granted (using verbs like `get`, `create`, `delete` etc.) for a set of resources. You can define these roles yourself, or get started with the default roles we provide.
```

Comment:
customers can't manage `ClusterRole`

Reply:
As in: no permissions for customer admins to add ClusterRole ?

Reply:
Ping @corest Can you clarify that please? Why can't customer admins manage ClusterRole resources?